<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadCreationWidget</class>
 <widget class="QWidget" name="UploadCreationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dashboard</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="lbl_supported_files">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Supported files are .gpkg, .zip or .csv</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lne_projection">
     <property name="text">
      <string>Select the SRS to use:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbl_files_put">
     <property name="text">
      <string>Pick a local dataset to import:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QgsProjectionSelectionWidget" name="psw_projection"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_data">
     <property name="text">
      <string>Name your dataset:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lne_dataset">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhLatinOnly|Qt::ImhUrlCharactersOnly</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" rowspan="2" colspan="2">
    <widget class="QTreeView" name="trv_upload_files"/>
   </item>
   <item row="1" column="1">
    <widget class="QgsFileWidget" name="flw_files_put">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhLatinOnly</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="lbl_files_summary">
     <property name="text">
      <string>List of datasets to import:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name your data</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="lne_data"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgsprojectionselectionwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
